src/pages/resources/migration/android/migrate-to-3x.md

```diff
@@ -99,3 +99,72 @@ Some of the APIs available in previous major versions of the Mobile SDK for Andr
 <InlineAlert variant="warning" slots="text"/>
 
 The `registerExtension` API for each extension that was deprecated in the 2.x version of the mobile SDK has been removed in the 3.x version of the mobile SDK. See the [Update SDK initialization](#update-sdk-initialization) section for more details.
+
+#### Edge Bridge
```

Comment:
I had it as a sub-element of updating outdated API references since (at least from my understanding), these Edge Bridge changes _are_ API changes right?

Reply:
It is intended as a subelement of "Update outdated API references" as that is where the docs are in the iOS migration guide. However, these not really API changes as only the format of the data in the request to the Edge Network has changed, and not the public APIs.

Reply:
I like "API migration and breaking changes"!

Reply:
cc @praveek do we need to update this for the other migration guides we have?

Reply:
This is the text earlier in the page. https://github.com/AdobeDocs/aep-mobile-sdkdocs/blob/migration-android/src/pages/resources/migration/android/migrate-to-3x.md?plain=1#L52

 The section "API migration and breaking changes" does not match the overall tone.